rocket/forms.py

```diff
@@ -23,3 +24,13 @@ class Meta:
             'last_name',
             'password',
         }
+
+class UserProfileForm(forms.ModelForm):
+    bio = forms.CharField(max_length=500)
+    entryno = forms.CharField(max_length=20)
```

Comment:
We don't want the user to input the entryno. Entry number should be fetched by us from the email ID.